modules/ROOT/pages/admin-center.adoc

```diff
@@ -100,7 +100,7 @@ The **Server Config** tool displays configuration files, such as a `server.xml`
 
 The **Design** mode displays the content of configuration files by using graphical controls with inline documentation. In this mode, select elements in the configuration to see enabled features and parameter settings.
 
-The **Source** mode enables direct access to edit the file text and provides tooltip descriptions for file elements. Hover on elements and parameters to view their documentation. You can press Ctrl+Space to use content assist, which helps you add new elements, parameters, and values. However, the **Source** mode editor does not validate the configuration values that you specify. Changes that you make in **Source** mode are applied to your running server and entering invalid configuration values might produce errors or cause your server to fail. 
+The **Source** mode enables direct access to edit the file text and provides tooltip descriptions for file elements. Hover on elements and parameters to view their documentation. You can press Ctrl+Space to use content assist, which helps you add new elements, parameters, and values. However, the **Source** mode editor does not validate the configuration values that you specify. Changes that you make in **Source** mode are applied to your running server and entering invalid configuration values might produce errors or cause your server to fail.
```

Comment:
Hover on elements and parameters to view their documentation. OR Hover **over** elements and parameters to view their documentation.

Changes that you make in **Source** mode are applied to your running server and entering invalid configuration values might produce errors or cause your server to fail.
or
Changes that you make in **Source** mode are applied to your running server and **specifying** invalid configuration values might produce errors or cause your server to fail.

Reply:
updated